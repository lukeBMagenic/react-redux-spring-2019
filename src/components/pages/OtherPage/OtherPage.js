import React, { Component } from 'react';
import ErrorBoundary from "../../@shared/ErrorBoundary/ErrorBoundary";
import PageHeader from "../../@shared/PageHeader/PageHeader";
import './OtherPage.scss';

class OtherPage extends Component {
  render() {
    return (
      <ErrorBoundary>
        <PageHeader />
        <h1>Other Page</h1>
      </ErrorBoundary>
    );
  }
}

export default OtherPage;
