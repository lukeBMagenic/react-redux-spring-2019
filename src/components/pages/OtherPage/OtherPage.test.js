import React from 'react';
import { shallow } from "enzyme";

import OtherPage from './OtherPage';

describe("OtherPage", () => {
  it("should render correctly", () => {
    const component = shallow(<OtherPage />);

    expect(component).toMatchSnapshot();
  });
});
