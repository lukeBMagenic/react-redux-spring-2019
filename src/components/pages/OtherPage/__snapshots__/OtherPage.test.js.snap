// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OtherPage should render correctly 1`] = `
<ErrorBoundary>
  <withRouter(WithStyles(PageHeader)) />
  <h1>
    Other Page
  </h1>
</ErrorBoundary>
`;
